capabilities/runtime.md

```diff
@@ -0,0 +1,22 @@
+---
+displayName: Serverless Runtime
+epicsLabels:
+  - area/serverless
+---
```

Comment:
With the new roadmap feature, please also add the id metadata:
```
id: "serverless"
```
so that the capability could display on the kyma-project.io roadmap.